README.md

```diff
@@ -2,6 +2,14 @@
 
 ffwdme.js is a JavaScript toolkit that aims to bring interactive GPS driving directions to the mobile browser.
 
+## Fork Reason
```

Comment:
No need to put this into the README, please remove. :)

Reply:
done.